### ISSUE TITLE
Proposal Feedback

### ISSUE THREAD

**Issue by stephenjarrell19** [COGS118A/Group001-Sp22#1]
Total Points: 7.25/8

Title & Abstract (0.75/1)
Rephrase/improve your abstract grammar and whatnot with something like: "We will be training logistic regression and random decision forest classifiers to classifier cardiovascular disease with factors such as smoking, alcohol use, or stroke history. By optimizing our models on this data set, we aim to develop a model that can assist in cardiovascular disease prevention and thus minimize the number of heart disease complications, which is the leading cause of death in the United States."

Your statement of "Performance will be measured by if the factors directly or indirectly influnce heart disease." is vague. Describe this in terms of what you learned in this course, whether that be metrics, distance formulas, or probabilities.

Background (1/1)
Nice!

Problem Statement (0.75/1)
How will you differentiate the difference between direct and indirect factors? Do you intend to? Are you doing any feature engineering to ensure that your features are not correlated or that they are not collinear? 

What techniques to balance class distribution?

What does "we will run this alongside our whole dataset" mean? Do you mean you are training a model and evaluating it on a test set of 400k unseen patients?

If you are using patient data, make sure your training set and test set does not include the same patient in your dataset.

Data (1/1)
Excellent description of your dataset. Can you reformat this section for legibility? Perhaps bullet points.

Proposed Solution (1/1)
How are these models different? Why might we want one over the other?

Metrics (0.75/1)
Use more than one metric. Using Precision? Use recall too. Or f1-score.

Ethics & Privacy (1/1)
This is good.

Team Expectations + Timeline (1/1)
Also good.

You can reply to this feedback below. Contact me anytime if you want help improving your project or have any questions at all!
